static/app/components/eventOrGroupHeader.tsx

```diff
@@ -89,14 +82,10 @@ function EventOrGroupHeader({
   function getTitle() {
     const {id, status} = data as Group;
     const {eventID: latestEventId, groupID} = data as Event;
-    const hasEscalatingIssues = organization.features.includes('escalating-issues');
 
     const commonEleProps = {
       'data-test-id': status === 'resolved' ? 'resolved-issue' : null,
-      style:
-        status === 'resolved' && !hasEscalatingIssues
-          ? {textDecoration: 'line-through'}
-          : undefined,
+      style: undefined,
```

Comment:
I also don't do much FE (lol), can we just remove the `style: undefined`? cc @scttcper